eox_nelp/third_party_auth/pipeline.py

```diff
@@ -94,8 +96,18 @@ def safer_associate_username_by_uid(  # pylint: disable=unused-argument
     if not user_match:
         return None
     if user_match.is_staff or user_match.is_superuser:
-        raise EoxNelpAuthException(backend, "It is not allowed to auto associate staff or admin users")
-
+        return HttpResponseForbidden(
+            edxmako.shortcuts.render_to_string(
+                "static_templates/server-error.html",
+                {
+                    "page_header": _("It is not allowed to auto associate staff or admin users"),
+                    "page_content": _(
+                        "You can change permission for that user or try to check the signupsource of the user."
```

Comment:
I'm sure about the relation of the signupsource model here,anyway I prefer something like "You are trying to access with a user that already exists. Please try to authenticate on the site where you registered for the first time or contact support."

Reply:
[a09575e](https://github.com/eduNEXT/eox-nelp/pull/113/commits/a09575e7c82931e92c6927f28575a7f32fa16048)
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/2291236e-1a9c-446b-a5e4-3eb932e78f0c)

Reply:
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/d70a5802-a7b9-4c87-b05c-95731acb1718)